slp_abacus/slp_abacus/validate/abacus_validator.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import os
+
+import xmlschema
+
+from sl_util.sl_util.file_utils import get_file_type_by_content
+from slp_base import ProviderValidator, DiagramFileNotValidError, DiagramType
+from slp_base.slp_base.provider_validator import generate_size_error, generate_content_type_error, generate_schema_error
+
+logger = logging.getLogger(__name__)
+path = os.path.dirname(__file__)
+
+class AbacusValidator(ProviderValidator):
+
+    def __init__(self):
+        super(AbacusValidator, self).__init__()
+
+
+    def validate(self):
```

Comment:
To comply with our security guidelines, we need to perform some validations to the source files. At least, we should validate that the file size is inside a certain range and that the content type is one of the expected ones. You can take as a reference the `DrawioValidator`. As for now, we can forget about validating any schema in this case.